### PR TITLE
[12.0][fix] base_tier_validation

### PR DESCRIPTION
**Pull request by ps-tubtim**
- show column comment on Reviews table

### CODE REVIEW

**Comment by ps-tubtim** [comment]
cc: @lreficent

**Comment by LoisRForgeFlow** [comment]
I missed the comment field displayed anywhere in your previous PR but I thought you were using it somewhere in your final implementation.

Once you doing some fixing this let me bring another issue to the spotlight: when you use a tier definition with "approve_sequence" set to true, you correctly hide the buttons to validate while a previous tier review is pending, but it is still shown in the systray counter as if you had a review pending. Could you take this opportunity to fix that? I promise I will be reviewing faster than last time :wink:

**Comment by ps-tubtim** [comment]
@lreficent I fixed about notification with "approve_sequence".

**Comment by ps-tubtim** [comment]
@lreficent I fixed about notification with "approve_sequence".

**Comment by LoisRForgeFlow** [comment]
When you click in the systray link you still get to a list with records that you can't validate yet:
image

**Comment by ps-tubtim** [comment]
@lreficent Edited 😊

**Review by LoisRForgeFlow — APPROVED** [review]
It looks great now! Thanks

**Comment by LoisRForgeFlow** [comment]
@kittiu can you review?

**Comment by kittiu** [comment]
LGTM :+1: 

But I would think we should review for better readme files, as we having more feature, i.e., comment, sequence.
Also, let's add the HISTORY.rst too.

@ps-tubtim can you do it? Will help review it.

**Review by kittiu — APPROVED** [review]

File: base_tier_validation/readme/HISTORY.rst [hunk 1]
```diff
@@ -0,0 +1,78 @@
+12.0.3.3.1 (2019-12-02)
```
Comment (kittiu) [inline base_tier_validation/readme/HISTORY.rst:1]: Great, I didn't think you will pull the whole history before what you did though :) But good to have them.

File: base_tier_validation/readme/CONFIGURE.rst [hunk 2]
```diff
@@ -3,3 +3,9 @@ To configure this module, you need to:
 #. Go to *Settings > Technical > Tier Validations > Tier Definition*.
 #. Create as many tiers as you want for any model having tier validation
    functionality.
+
+**Note:**
+
+* If check **Notify Reviewers on Creation**, all possible reviewers will be notified by email when this definition is triggered.
+* If check **Comment**, reviewers can comment after click Validate or Reject.
+* If check **Approve by sequence**, reviewers must be review respectively.
```
Comment (kittiu) [inline base_tier_validation/readme/CONFIGURE.rst]:
```suggestion
* If check **Approve by sequence**, reviewers is forced to review by specified sequence.
```

**Review by kittiu — APPROVED** [review]

File: base_tier_validation/readme/CONFIGURE.rst (hunk 2 above)
Comment (ps-tubtim) [reply base_tier_validation/readme/CONFIGURE.rst]: Edited.

**Pull request merged**